### PR TITLE
Implement fixes from https://github.com/uBlockOrigin/uAssets/blob/master/filters/lan-block.txt#L107

### PR DESCRIPTION
**Pull request by ryanbr**
Implement upstream localhost fixes from https://github.com/uBlockOrigin/uAssets/blob/master/filters/lan-block.txt#L107

A copy/paste with comments in https://github.com/uBlockOrigin/uAssets/blob/master/filters/lan-block.txt#L107

### CODE REVIEW

**Comment by ryanbr** [comment]
Hmm I wonder if there is any interaction with https://github.com/brave/adblock-lists/blob/master/brave-lists/localhost-permission-allow-list.txt

**Comment by ryanbr** [comment]
I'll can this, and implement via https://github.com/brave/adblock-lists/blob/master/brave-lists/localhost-permission-allow-list.txt

**Comment by ShivanKaul** [comment]
That list is only used by the localhost permission feature, which is only enabled in Nightly (brave://flags/#brave-localhost-access-permission)

**Comment by ryanbr** [comment]
Okay, we could do both, re-opened

File: brave-lists/brave-specific.txt [hunk 1]
```diff
@@ -36,6 +36,33 @@ search.brave.com###search-ad
 ! https://community.brave.com/t/bug-brave-shields-break-trezor-bridge-connect-when-accessing-wallet/219544/
 @@||127.0.0.1^$domain=trezor.io
 
+!!! START
+! https://github.com/uBlockOrigin/uAssets/blob/master/filters/lan-block.txt#L107
```
Comment (fmarier) [inline brave-lists/brave-specific.txt:40]: Let's use a link to the issue, not the list:
```suggestion
! https://github.com/uBlockOrigin/uAssets/issues/23456
```

File: brave-lists/brave-specific.txt [hunk 2]
```diff
@@ -36,6 +36,33 @@ search.brave.com###search-ad
 ! https://community.brave.com/t/bug-brave-shields-break-trezor-bridge-connect-when-accessing-wallet/219544/
 @@||127.0.0.1^$domain=trezor.io
 
+!!! START
```
Comment (fmarier) [inline brave-lists/brave-specific.txt:39]:
```suggestion
!!! START OF UBO EXCEPTIONS
```

File: brave-lists/brave-specific.txt [hunk 3]
```diff
@@ -36,6 +36,33 @@ search.brave.com###search-ad
...
+! https://github.com/uBlockOrigin/uAssets/commit/b0e28ffe
+@@||127.0.0.1^$domain=intel.cn|intel.co.id|intel.co.jp|intel.co.kr|intel.com|intel.com.br|intel.com.tw|intel.de|intel.fr|intel.la|intel.vn
+!
+! https://github.com/uBlockOrigin/uAssets/issues/23388
+@@||localhost^$domain=deutsche-rentenversicherung.de
+@@||127.0.0.1^$domain=bund.de|organspende-register.de|bayernportal.de|mv-serviceportal.de
+!
+! https://github.com/uBlockOrigin/uAssets/pull/24488
+@@||localhost^$websocket,domain=www.faceit.com
+!!! END
```
Comment (fmarier) [inline brave-lists/brave-specific.txt:64]:
```suggestion
!!! END OF UBO EXCEPTIONS
```

**Review by fmarier — APPROVED** [review]

**Pull request merged**